oonimkall/runner.go

```diff
@@ -47,7 +47,7 @@ func newRunner(settings *settingsRecord, out chan<- *eventRecord) *runner {
 	}
 }
 
-func (r *runner) hasUnsupportedSettings() (unsupported bool) {
+func (r *runner) hasUnsupportedSettings(logger *chanLogger) (unsupported bool) {
```

Comment:
A bunch of cases have become warnings because otherwise it's complex to keep MK/non-MK code together. We may want to consider removing these fields in the future, maybe using something like an API number to kick out old clients.